resolving/npm-resolver/src/index.ts

```diff
@@ -173,7 +175,7 @@ async function resolveNpm (
           wantedDependency,
           projectDir: opts.projectDir,
           lockfileDir: opts.lockfileDir,
-          hardLinkLocalPackages: wantedDependency.injected,
+          hardLinkLocalPackages: opts.injectWorkspacePackages === true || wantedDependency.injected,
```

Comment:
I have a question, if `opts.injectWorkspacePackages` is set to `false`, should it have higher priority than `wantedDependency.injected`?